test/1_test_fullpath.js

```diff
@@ -292,16 +313,21 @@ contract('Voucher tests', (addresses) => {
       );
 
       //Check ERC1155ERC721 state
-      const sellerERC1155ERC721Balance = await contractERC1155ERC721.balanceOf(
-        users.seller.address,
-        tokenSupplyKey1
-      );
+      const sellerERC1155ERC721Balance = (
+        await contractERC1155ERC721.functions[fnSignatures.balanceOf1155](
```

Comment:
Same question as above. Why is the function signature being used instead of calling the function directly?